docs/12-upgrading/01-upgrade-to-two.md

```diff
@@ -212,4 +212,36 @@ The authentication module's default value for allowing unsecure random number ge
 auth.AuthConfig.set(auth.AuthConfig(
   allowUnsecureRandom: true,
 ));
-```
\ No newline at end of file
+```
+
+## Updates to Custom Serialization in Serverpod 2.0
```

Comment:
Suggestion: This update changes all Model Serialization not just Custom Serialization since the parameter is not required when serializing Serverpod Models as well.

Please also describe that models can now be deserialized by calling `fromJson(...)` without the serialization manager parameter.

But I think it is good to mention how it effects the custom serialization as well. So these should probably be two separate sections.